File: HelloWorldExample/ViewController.m
//
//  ViewController.m
//  HelloWorldExample
//
//  Created by peter.shih on 2019/7/23.
//  Copyright © 2019年 Peter.shih. All rights reserved.
//

#import "ViewController.h"
#import "HelloWorldSDK/HelloWorldSDK.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UILabel *titleLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    _titleLabel.text = [HelloWorld helloWithName:@"小風小馬"];
}


@end

File: HelloWorldExampleApp/ViewController.swift
//
//  ViewController.swift
//  HelloWorldExampleApp
//
//  Created by peter.shih on 2019/8/27.
//  Copyright © 2019年 Peteranny. All rights reserved.
//

import UIKit
import HelloWorldSDK

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let label: UILabel = {
            let label = UILabel()
            label.frame.origin = CGPoint(x: 100, y: 100)
            label.text = HelloWorld().hello(to: "World")
            label.textColor = HelloWorld().helloColor
            label.sizeToFit()
            return label
        }()
        
        view.addSubview(label)
    }
    
}
